docs/docs-components/data/components.js

```diff
@@ -852,6 +852,15 @@ const componentData: $ReadOnlyArray<ComponentData> = [
           figmaStatus: 'ready',
           responsive: 'ready',
           status: 'ready',
+          knownIssues: [
+            {
+              title: 'Line clamp truncation',
+              description:
+                'When "lineClamp" prop is set to 1, Heading truncates  strings containing long strings of characters incorrectly.',
```

Comment:
This doesn't read as complete sentence, maybe:

When the "lineClamp" prop is set to 1, text with long strings of characters maybe truncated incorrectly

Reply:
DONE